lang/en.js

```diff
@@ -26,13 +26,13 @@ export default {
 	"components.form-element.defaultError": "{label} is invalid.",
 	"components.form-element.defaultFieldLabel": "Field",
 	"components.form-element.input.email.typeMismatch": "Email is not valid",
-	"components.form-element.input.number.rangeError": "Number must be {minExclusive, select, true {greater than} false {greater than or equal to}} {min} and {maxExclusive, select, true {less than} false {less than or equal to}} {max}.",
-	"components.form-element.input.number.rangeOverflow": "Number must be {maxExclusive, select, true {less than} false {less than or equal to}} {max}.",
-	"components.form-element.input.number.rangeUnderflow": "Number must be {minExclusive, select, true {greater than} false {greater than or equal to}} {min}.",
+	"components.form-element.input.number.rangeError": "{minExclusive, select, true {{maxExclusive, select, true {Number must be greater than {min} and less than {max}.} other {Number must be greater than {min} and less than or equal to {max}.}}} other {{maxExclusive, select, true {Number must be greater than or equal to {min} and less than {max}.} other {Number must be greater than or equal to {min} and less than or equal to {max}.}}}}",
```

Comment:
For `rangeError`, it was hard enough to do the English one so I'm sending all the other languages back for re-translation. The others were easier so I was mostly confident I could "fix" them in the other languages.